index.html

```diff
@@ -23,22 +23,147 @@ <h1>Hey There.i'm Said</h1>
+        </div>
+        <ul class="tags">
+          <li class="css">css</li>
+          <li class="html">html</li>
+          <li class="boot">bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+        <form class="check">
+          <button type="button" class="click">See Project</button>
+        </form>
+      </section>
+      <section class="work project1">
+        <div class="linfo">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="lang">
+            <li class="ht">html</li>
+            <li class="bo">bootstrap</li>
+            <li class="ru">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="but">See Project</button>
+        </form>
+      </section>
+      <section class="work projectq">
+        <div class="linfoq">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="langq">
+            <li class="htq">html</li>
+            <li class="boq">bootstrap</li>
+            <li class="ruq">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="butq">See Project</button>
+        </form>
+      </section>
+      <section class="work projectw">
+        <div class="linfow">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="langw">
+            <li class="htw">html</li>
+            <li class="bow">bootstrap</li>
+            <li class="ruw">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="butw">See Project</button>
+        </form>
+      </section>
+      <section class="work projectX">
+        <div class="linfoe">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="leng">
+            <li class="hte">html</li>
+            <li class="strap">bootstrap</li>
+            <li class="rube">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="bute">See Project</button>
+        </form>
+      </section>
+      <section class="work about">
+        <h2>About me</h2>
+        <p class="par">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
```

Comment:
- [ ] You should stick to the figma design on the space between the heading the paragraph. It seems it is a bit wider on your design as compared to the design.
#### Design
![image](https://user-images.githubusercontent.com/44057863/170739581-1bf3b525-cee5-432f-a250-efaa56b0f40d.png)
#### Impl
![image](https://user-images.githubusercontent.com/44057863/170739516-89c41ad1-065c-43af-832e-412c158d5584.png)

Reply:
done

Reply:
Kindly update your design according to the previous reviewer recommendation. The margin between your `About me` is too large. 
|Figma Design|Your Design|
|---|---|
|![image](https://user-images.githubusercontent.com/12239821/170779909-36277586-b774-481f-84b6-c232c7426eb4.png)|![image](https://user-images.githubusercontent.com/12239821/170779684-b210dcdd-a98e-48c7-b523-a4a13a958b17.png)|

Reply:
done @chrystalme